### PR TITLE
#1733 Redistribute payout

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/modals/tx-sender/investor-payout/RedistributeConfirm.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+type IProps = IDispatchProps;
+
+const InvestorRedistributePayoutConfirmLayout: React.FunctionComponent<IProps> = ({
+  deny,
+  confirm,
+}) => (
+  <Message
+    image={<ConfettiEthereum className="mb-3" />}
```
Comment (desfero) [inline app/components/modals/tx-sender/investor-payout/RedistributeConfirm.tsx]: Will replace by proper icon after @falinebambi provide one

File: app/components/modals/tx-sender/investor-payout/RedistributeConfirm.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { action } from "@storybook/addon-actions";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { withModalBody } from "../../../../utils/storybookHelpers";
```
Comment (drakkein) [inline app/components/modals/tx-sender/investor-payout/RedistributeConfirm.stories.tsx]: Please split it to 2 groups

File: app/components/modals/tx-sender/investor-payout/RedistributeConfirm.tsx (hunk 1 above)
Comment (drakkein) [reply app/components/modals/tx-sender/investor-payout/RedistributeConfirm.tsx]: 👌

File: app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { action } from "@storybook/addon-actions";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { withModalBody } from "../../../../utils/storybookHelpers";
```
Comment (drakkein) [inline app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx]: Please split it to 2 groups

File: app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        <InfoRow
+          key={tokenDisbursal.token}
+          caption={
+            <FormattedMessage id="investor-payout.redistribute.summary.total-redistributed" />
+          }
+          value={<Money value={tokenDisbursal.amountToBeClaimed} currency={tokenDisbursal.token} />}
+        />
+      </InfoList>
+      <section className="text-center">
+        <a
```
Comment (drakkein) [inline app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx]: Can we use ExternalLink component here?

File: app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+import { InfoList } from "../shared/InfoList";
+import { InfoRow } from "../shared/InfoRow";
+
+interface IStateProps {
+  tokenDisbursal: ITokenDisbursal;
+  walletAddress: EthereumAddressWithChecksum;
+}
+
+interface IDispatchProps {
+  onAccept: () => any;
```
Comment (drakkein) [inline app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx]: can we use `void` typing here?

File: app/modules/tx/sender/actions.ts [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
 import { ETransactionErrorType, TSummaryData } from "./reducer";
 
 export const txSenderActions = {
   // Modal related actions
   txSenderShowModal: (type: ETxSenderType) => createAction("TX_SENDER_SHOW_MODAL", { type }),
   txSenderHideModal: () => createSimpleAction("TX_SENDER_HIDE_MODAL"),
   // User awaiting actions
-  txSenderAcceptDraft: (txDraftData?: Partial<ITxData>) =>
-    createAction("TX_SENDER_ACCEPT_DRAFT", { txDraftData }),
+  txSenderAcceptDraft: createActionFactory(
```
Comment (Moejoe90) [inline app/modules/tx/sender/actions.ts]: :+1:

File: app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx (hunk 5 above)
Comment (Moejoe90) [reply app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx]: :+1:

File: app/components/modals/tx-sender/investor-payout/RedistributeConfirm.stories.tsx (hunk 2 above)
Comment (desfero) [reply app/components/modals/tx-sender/investor-payout/RedistributeConfirm.stories.tsx]: done

File: app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx (hunk 3 above)
Comment (desfero) [reply app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx]: done

File: app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx (hunk 4 above)
Comment (desfero) [reply app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx]: done

File: app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx (hunk 5 above)
Comment (desfero) [reply app/components/modals/tx-sender/investor-payout/RedistributeSummary.tsx]: done

File: app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/modals/tx-sender/investor-payout/RedistributeSuccess.stories.tsx]: done

**Comment by Desperado** [comment]
Looks quite good!

**Pull request merged**